extensions/in-memory/policy-store-memory/src/test/java/org/eclipse/dataspaceconnector/policy/store/memory/InMemoryPolicyStoreTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.policy.store.memory;
+
+import org.eclipse.dataspaceconnector.policy.model.Policy;
+import org.eclipse.dataspaceconnector.spi.contract.policy.store.PolicyStore;
+import org.eclipse.dataspaceconnector.spi.query.QuerySpec;
+import org.eclipse.dataspaceconnector.spi.query.SortOrder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InMemoryPolicyStoreTest {
+    private PolicyStore store;
+
+    @BeforeEach
+    void setUp() {
+        store = new InMemoryPolicyStore();
+    }
+
+    @Test
+    void findById_whenPresent() {
+        Policy policy = Policy.Builder.newInstance().build();
+        store.save(policy);
+
+        assertThat(store.findById(policy.getUid())).isEqualTo(policy);
+    }
+
+    @Test
+    void findById_whenMissing() {
+        assertThat(store.findById(UUID.randomUUID().toString())).isNull();
```

Comment:
nit: From a code review I got from Paul I think that their personal preference is to use an id called something like "not-exists" for this type of tests where you query/delete a non existing item.

Reply:
ok, thanks, will change that